File: SecondYear/Vaishnavi0-0_VaishnaviSingh_2226cseml1114_2/week_1/inputoutput.java
import java.util.Scanner;

public class inputoutput {
    public static void main(String[] args) {
        try (// Write your code here
                Scanner read = new Scanner(System.in)) {
            char ch = read.next().charAt(0);
            if (ch >= 'a' && ch <= 'z')
                System.out.println(0);
            else if (ch >= 'A' && ch <= 'Z')
                System.out.println(1);
            else
                System.out.println(-1);
        }

    }
}

File: SecondYear/anoushka-10_AnoushkaGoel_2226cseml1134_2/Week_1/inputoutput.c
#include<stdio.h>


int main() {
	// Write your code here
	char a;
	scanf("%c",&a);
	if(a>='a' && a<='z'){
		printf("0");
	}
	if(a>='A' && a<='Z'){
		printf("1");
	}
	else{
		printf("-1");
	}
	return 0;
}
